lms/djangoapps/courseware/views.py

```diff
@@ -638,9 +638,23 @@ def jump_to(_request, course_id, location):
     elif section is None:
         return redirect('courseware_chapter', course_id=course_key.to_deprecated_string(), chapter=chapter)
     elif position is None:
-        return redirect('courseware_section', course_id=course_key.to_deprecated_string(), chapter=chapter, section=section)
+        return redirect(
+            'courseware_section',
+            course_id=course_key.to_deprecated_string(),
```

Comment:
Nit: since you're changing this line, can you update it to use `unicode(course_key)`? The functionality is identical.